Issue #346: App Redesign: Account Modal

jahabeebs opened the pull request:
Closes #346 

## Description
- Reskins the account modal and the view/top-up other vaults modals to match the styles of the rest of the app

## Screenshots

Desktop 

<img width="985" alt="desktop" src="https://github.com/open-dollar/od-app/assets/47253537/cc035f34-bedc-453d-8607-44ee13c103e3">

Mobile

<img width="471" alt="mobile" src="https://github.com/open-dollar/od-app/assets/47253537/97610830-9b3a-45b1-9e13-d0ed3ffbac36">

Top-up desktop

<img width="839" alt="top up desktop" src="https://github.com/open-dollar/od-app/assets/47253537/dbff8b21-cc21-44e7-9bcc-83f316cab7b0">

Top-up mobile

<img width="472" alt="top up mobile" src="https://github.com/open-dollar/od-app/assets/47253537/90c91537-a4f4-4f95-beba-83ab5d7186d0">

pi0neerpat reviewed (changes requested):
- [ ] Account icon for coinbase alignment

![image](https://github.com/open-dollar/od-app/assets/35622595/6afdfdef-7834-4808-8a3e-6caf8723f9d6)

- [ ] Font color for transactions needs updating

![image](https://github.com/open-dollar/od-app/assets/35622595/8cba771e-aa9b-41be-a093-de6a17ce9852)

jahabeebs commented:
> * [ ]  Account icon for coinbase alignment
> 
> 
> ![image](https://private-user-images.githubusercontent.com/35622595/322170536-6afdfdef-7834-4808-8a3e-6caf8723f9d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyMDcwNzgsIm5iZiI6MTcxMzIwNjc3OCwicGF0aCI6Ii8zNTYyMjU5NS8zMjIxNzA1MzYtNmFmZGZkZWYtNzgzNC00ODA4LThhM2UtNmNhZjg3MjNmOWQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDE4NDYxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3Y2VhYmViMDI3Y2M1YTUyYjE2YzgwMzI1OTMzZTk3MTE3ZWNkZDk2MmY2ZjY4NWVlMGY4ZTU2NDJjMzEwZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MHo-1yzwhggKKRunyIF3RQ5SdcDtVbh6KKCRSavzN3U)
> 
>     * [ ]  Font color for transactions needs updating
> 
> 
> ![image](https://private-user-images.githubusercontent.com/35622595/322170566-8cba771e-aa9b-41be-a093-de6a17ce9852.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyMDcwNzgsIm5iZiI6MTcxMzIwNjc3OCwicGF0aCI6Ii8zNTYyMjU5NS8zMjIxNzA1NjYtOGNiYTc3MWUtYWE5Yi00MWJlLWEwOTMtZGU2YTE3Y2U5ODUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDE4NDYxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBhMzIxMDY3NTQ2N2IwNTVhZTMxZDg2ZDZiOTg0ZDFlMTUwNGVhMTliNzY2ZTIwMGY2YjEzZDIyOWUxMzVlZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.onH7piLfXd8Lj3CX2CPnIEKrgeVSxxUBTaITa3gfYBc)

- Changed recent transaction text to white
- Fixed icon alignment issue of the identicon by reducing the line height of the container to 20px

## Screenshots

<img width="745" alt="fix account modal bugs" src="https://github.com/open-dollar/od-app/assets/47253537/39d155c3-ca46-43ea-9ae4-7aa9ed8aa6ab">
